src/kv/untyped_map_handle.h

```diff
@@ -82,16 +83,23 @@ namespace kv::untyped
     }
 
   public:
-    MapHandle(ChangeSet& cs) : tx_changes(cs) {}
+    MapHandle(ChangeSet& cs, const std::string& map_name) :
+      tx_changes(cs),
+      map_name(map_name)
+    {}
 
     std::optional<ValueType> get(const KeyType& key)
     {
       auto value_p = read_key(key);
-      if (value_p == nullptr)
+      auto found = value_p != nullptr;
+#ifdef VERBOSE_LOGGING
```

Comment:
These don't need to be guarded - instead the internals of `LOG_TRACE_FMT` will be enabled when `VERBOSE_LOGGING` is on (but potentially in future also from other things).

Reply:
I want to avoid constructing the string etc., even if it is not logged. `VERBOSE_LOGGING` is only used to determine the logging level currently, it should probably also make `LOG_TRACE` a no-op in that case.

Reply:
I removed the guards and we'll do this in the logging macro directly (separate PR).